airflow/providers/atlassian/jira/operators/jira.py

```diff
@@ -76,16 +76,11 @@ def execute(self, context: Context) -> Any:
                 hook = JiraHook(jira_conn_id=self.jira_conn_id)
                 resource = hook.client
 
-            # Current Jira-Python SDK (1.0.7) has issue with pickling the jira response.
-            # ex: self.xcom_push(context, key='operator_response', value=jira_response)
-            # This could potentially throw error if jira_result is not picklable
             jira_result = getattr(resource, self.method_name)(**self.jira_method_args)
             if self.result_processor:
                 return self.result_processor(context, jira_result)
```

Comment:
If we remove the notes since no longer relevant shouldn't the code change as well?
to my understanding this code is a workaround to address the issue in comments

Reply:
The official SDK returns 'id' in the output in all jira issue related methods. I have added these https://github.com/apache/airflow/pull/27633/files#diff-85731758409b94fa7ff2ab57860f1d4242f7a196768f003e7b618d88254afba4R84-R85 which takes id from the response and performs `xcom_push` with that.